### PR TITLE
Gdr 2316

### PR DESCRIPTION
**Pull request by bczech**
# Description
## What changed?
Related JIRA issue: GDR-2316

## Why was it changed?

# Checklist for sustainable code base
- [ ] I added tests for any code changed/added
- [ ] I added documentation for any code changed/added
- [ ] I made sure naming of any new functions is self-explanatory and consistent

# Logistic checklist
- [ ] Package version bumped
- [ ] Changelog updated

# Screenshots (optional)

### CODE REVIEW

**Comment by gladkia** [comment]
Please bump version and update changelog.

**Review by gladkia — APPROVED** [review]

File: R/map_df.R [hunk 1]
```diff
@@ -250,15 +249,50 @@ map_df <- function(trt_md,
...
+#' 
+#' @param mat_elem input data frame
+#'
+#' @details
+#' Using the given rownames, map the untreated conditions
+#' 
+#' @return list
+#' 
+map_untreated <- function(mat_elem) {
+  # TODO: avoid recycling code of map_references above
```
Comment (j-smola) [inline R/map_df.R:276]: Could it be described as jira task?

**Review by j-smola — APPROVED** [review]

File: R/map_df.R (hunk 1 above)
Comment (bczech) [reply R/map_df.R:276]: Done. [GDR-2332](https://jira.gene.com/jira/browse/GDR-2332)

File: tests/testthat/test-map_df.R [hunk 2]
```diff
@@ -33,19 +33,26 @@ test_that("map_df works as expected", {
 # TODO: test_that("NAs are returned for missing controls", {}) # nolint
```
[the remark this answers is not in the document]
Comment (bczech) [reply tests/testthat/test-map_df.R:34]: Done. [GDR-2333](https://jira.gene.com/jira/browse/GDR-2333)

**Pull request merged**